capstone/src/main/java/com/google/sps/data/ReadData.java

```diff
@@ -0,0 +1,72 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.sps.data.SurveyResponse;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.lang.NullPointerException;
+
+public class ReadData {
+    final String DELIMITER = ",";
+
+    public void readCSV() {
+        SurveyResponse response;
+        BufferedReader reader;
+        FileReader file;
+        DatastoreService datastoreService = DatastoreServiceFactory.getDatastoreService();
+
+        try {
+            file = new FileReader("assets/surveyresponse.csv");
+            reader = new BufferedReader(file);
+            String header = reader.readLine();
+            String line = "";
+
+            while ((line = reader.readLine()) != null) {
+                String[] values = line.split(DELIMITER); 
+                String id = values[0];
```

Comment:
Is this the Survey Id or User Id? You probably want to set the id to something like 

<Survey Id>-<User Id>

That way you don't ignore responses for each survey, or from duplicate users on different surveys

Reply:
Btw I'm assuming 1 response here captures all the different question answers into a single entity.

Reply:
Currently, it's the Survey ID.  Is each survey ID unique?

Reply:
At the moment, I set up the code under the assumption that I'm only considering one free text.  Since we have more survey questions I plan to add more fields to accommodate for each question

Reply:
Each Survey ID is unique but each Survey ID has multiple Questions. Also, each Survey ID has multiple respondents (User ID). If you want to store the entity ID at the Survey ID level, that means you'd have to store all the responses from each User ID in an array-like field or something.

Otherwise, you can have the ID be something like `<Survey ID>-<User ID>`, which means there is one entity per survey response as opposed to one entity per survey

Reply:
And then, each response captures multiple `Question ID`s too. So if you wanted to store one entity per survey response to each question, you would want an ID like `<Survey ID>-<User ID>-<Question ID>`. Make sense?

Reply:
I understand now, is it alright if I save that change for my next PR which is meant to make the synthetic results more accurate with the actual results?